nion/swift/Inspector.py

```diff
@@ -594,7 +594,7 @@ def __init__(self, document_controller: DocumentController.DocumentController, d
         self._display_item = display_item
         self._title_model = Model.PropertyChangedPropertyModel[str](display_item, "title")
         self._placeholder_title_model = Model.PropertyChangedPropertyModel[str](display_item, "placeholder_title")
```

Comment:
`self._title_model` needs to be changed too.

Reply:
@cmeyer I am guessing you missed my earlier comment: I tried doing this, but it failed: I got a conflict making a DisplayPanel.ChangeDisplayCommand because it already has a (different) title input giving me the error "nion.swift.DisplayPanel.ChangeDisplayCommand() got multiple values for keyword argument 'title'" Since the property/value pairs are passed into kwargs, it detected this as a second attempt to set the title, and I am not sure of the best way to resolve this.

Reply:
Yes, missed it! - see my explanation above.